gym_roboy/envs/roboy_env.py

```diff
@@ -137,3 +129,18 @@ def _did_complete_successfully(self, current_state: RobotState, goal_state: Robo
         if angles_are_close and vels_are_close:
             print("#############GOAL REACHED#############")
         return angles_are_close and vels_are_close
+
+
+def _l2_distance(joint_angle1, joint_angle2):
+    subtract = np.subtract(joint_angle1, joint_angle2)
+    subtract[np.isnan(subtract)] = 0  # np.inf - np.inf returns np.nan, but should be 0
+    return np.linalg.norm(subtract, ord=2)
+
+
+@typechecked
+def _rescale_from_one_space_to_other(
```

Comment:
Can you explain what you do here

Reply:
@BarisYazici I added documentation for the function